### ISSUE TITLE
Error in upgrade to v3.6.2

### ISSUE THREAD

**Issue by ewout** [academic-moodle-cooperation/moodle-mod_offlinequiz#110]
I got an error which I could resolve as follows (hopefully the code works as intended)

```
--- a/mod/offlinequiz/db/upgrade.php
+++ b/mod/offlinequiz/db/upgrade.php
@@ -1434,10 +1434,11 @@ function xmldb_offlinequiz_upgrade($oldversion = 0) {
         if ($idstodelete) {
         $chunks = array_chunk($idstodelete, 250, true);
         echo "Delete choices in ".count($chunks)." chunks of 250 entries...".PHP_EOL;
+        $i = 1;
         foreach ($chunks as $curchunk) {
             echo "Delete chunk ".($i++)." of ".count($chunks)."!".PHP_EOL;
             list($querysql, $queryparams) = $DB->get_in_or_equal($curchunk);
-            $DB->delete_records_select('offlinequiz_choices', 'id', $querysql, $queryparams);
+            $DB->delete_records_select('offlinequiz_choices', 'id '. $querysql, $queryparams);
         }
         }

```

**Comment by goggo24** [comment]
Thanks for reporting. We will have a look.

**Comment by goggo24** [comment]
Fix will be included in v.3.6.3+ and v.3.7.0+ - coming soon!

**Comment by goggo24** [comment]
3.6.3. is published, 3.7.0 will be published within the next days ;)
